apps/portals/ampals/src/pages/HomePage.tsx

```diff
@@ -4,11 +4,31 @@ import AMPALSHeader from '@sage-bionetworks/synapse-portal-framework/components/
 // import { FeaturedDataTabs } from 'synapse-react-client'
 // import columnAliases from '../config/columnAliases'
 import headerSvg from '../config/style/header.svg?url'
-import { FullWidthAlert } from 'synapse-react-client'
+import { FullWidthAlert, MoreResources } from 'synapse-react-client'
 import { OrientationBanner } from 'synapse-react-client'
 
 //TODO
 export default function HomePage() {
+  const resources = [
```

Comment:
Please rename to `moreResourcesContent` to avoid confusion with the code in resources.ts